src/util/paymentMethodHelper.js

```diff
@@ -2,6 +2,9 @@ import { ReactComponent as CardLogo } from 'assets/images/paymentMethods/card.sv
 import { ReactComponent as PaypalLogo } from 'assets/images/paymentMethods/PPicon.svg';
 import { ReactComponent as ApplePayLogo } from 'assets/images/paymentMethods/applePay.svg';
 import { ReactComponent as GooglePayLogo } from 'assets/images/paymentMethods/googlepay.svg';
+import { ReactComponent as idealLogo } from 'assets/images/paymentMethods/ideal-small.svg';
```

Comment:
Maybe to be consistent we can start with capital letter because we are importing thise svg as components?

Reply:
good point, fixed